index.html

```diff
@@ -204,55 +205,2367 @@ <h3>
+              task source=] to [=reject=] |promise| with a "{{NetworkError}}"
+              {{DOMException}} and abort these steps.
+              </li>
+              <li>[=Queue a global task=] on the [=relevant global object=] of
+              [=this=] using the [=HID device task source=] to [=resolve=]
+              |promise| with `undefined`.
+              </li>
+            </ol>
+          </li>
+          <li>Return |promise|.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          <dfn>receiveFeatureReport()</dfn> method
+        </h3>
+        <div class="note">
+          <p>
+            The {{HIDDevice/receiveFeatureReport()}} method is used to request
+            a feature report with the specified |reportId:octet|. If the [=HID
+            interface=] does not [=use report IDs=], pass 0 instead of the
+            report ID.
+          </p>
+          <p>
+            For [=HID interfaces=] that [=use report IDs=], the report ID value
+            0 is reserved and should not be used.
+          </p>
+        </div>
+        <p>
+          The {{HIDDevice/receiveFeatureReport()}} method steps are:
+        </p>
+        <ol>
+          <li>Let |promise:Promise| be [=a new promise=].
+          </li>
+          <li>If [=this=].{{HIDDevice/[[state]]}} is not `"opened"`, then
+          [=reject=] |promise| with a "{{InvalidStateError}}" {{DOMException}}
+          and return |promise|.
+          </li>
+          <li>If |reportId:octet| is 0 and the [=HID interface=] represented by
+          [=this=] [=uses report IDs=], or if |reportId| is not 0 and the
+          interface does not [=use report IDs=], then [=reject=] |promise| with
+          a "{{TypeError}}" {{DOMException}} and return |promise|.
+          </li>
+          <li>Append |promise| to
+          [=this=].{{HIDDevice/[[pendingReceiveFeatureReportPromises]]}}.
+          </li>
+          <li>Perform the following steps [=in parallel=]:
+            <ol>
+              <li>If |promise| is not in
+              [=this=].{{HIDDevice/[[pendingReceiveFeatureReportPromises]]}},
+              then abort these steps.
+              </li>
+              <li>Remove |promise| from
+              [=this=].{{HIDDevice/[[pendingReceiveFeatureReportPromises]]}}.
+              </li>
+              <li>Let |data:DataView| be the result of invoking the operating
+              system to read a feature report with the specified
+              |reportId:octet|.
+              </li>
+              <li>If this report is a [=blocked report=], [=queue a global
+              task=] on the [=relevant global object=] of [=this=] using the
+              [=HID device task source=] to [=reject=] |promise| with a
+              "{{NotAllowedError}}" {{DOMException}} and abort these steps.
+              </li>
+              <li>If this fails for any reason, [=queue a global task=] on the
```

Comment:
If what fails? I think you accidentally removed a step. 😄

Reply:
Oops, fixed